rollup.examples.config.js

```diff
@@ -0,0 +1,16 @@
+import nodeResolve from '@rollup/plugin-node-resolve';
+import styles from 'rollup-plugin-styles';
+
+const globals = id =>
```

Comment:
I copied this line from rollup.src.config because it fixed a warning or error message that I saw from the `rollup` command.